docs/md/man1/openroad.md

```diff
@@ -0,0 +1,112 @@
+---
+title: openroad(0)
+date: 2024/02/09
+---
+
+# NAME
+
+openroad - OpenROAD command-line-interface.
+
+# SYNOPSIS
+
+openroad [-help] [-version] [-no_init] [-exit] [-gui]
+       [-threads count|max] [-log file_name] 
+       [-metrics file_name] [-python]
+       cmd_file
```

Comment:
Missing -no_splash

Reply:
`-no_splash` is currently not showing when we do openroad -help. Is this intended?

Reply:
no, please fix it